requirements.txt

```diff
@@ -5,6 +5,7 @@ sentencepiece  # Required for LLaMA tokenizer.
 numpy
 torch == 2.1.2
 transformers >= 4.38.0  # Required for Gemma.
+hf_transfer==0.1.5 # for faster weight downloads
```

Comment:
Instead of pinning it here, can you change Docker file to install it in our Docker images?

Reply:
Also, why don't we use `>=` instead of pinning the version?

Reply:
does not it require installing rust for this package to work?

Reply:
> does not it require installing rust for this package to work?

@RonanKMcGovern @simon-mo Is this true?